lms/templates/learner_dashboard/program_card.underscore

```diff
@@ -0,0 +1,22 @@
+<a href="<%- marketingUrl %>" class="card-link">
+	<div class="banner-image">
+	</div>
+</a>
+<div class="text-section">
+	<div class="meta-info">
+		<div class="organization">
+			<% _.each(organizations, function(org){ %>
+				<span><%- gettext(org.display_name) %></span>
```

Comment:
Does each name need to be in its own ```<span>```? If not the code could be simplified to be ```_.pluck(organizations, 'display_name').join(' ');```